src/docs.md

```diff
@@ -48,25 +58,25 @@ here are some next steps.
 
 Once you’ve mastered the basics, try these pages.
 
-* [Cookbook](/cookbook/)<br>
+* [Cookbook](/cookbook)<br>
```

Comment:
Soon (it can be in a followup PR) we should get rid of the `<br>` elements and use proper styling. Maybe these entries need to be formatted as a markdown "dictionary"? /cc @fertolg